src/stores/slice/categoriesSlice.js

```diff
@@ -0,0 +1,41 @@
+import { createSlice,createAsyncThunk } from "@reduxjs/toolkit";
+import { ApiClient } from "../../request/request";
+
+const cateSlice=createSlice({
+    name:'category',
+    initialState:{
+        list:[],
+        meta:[],
+        error: null,
+    },
+    reducers:{
+        // getCategories:(state,action)=>{
+        //     state.items=action.payload
+        // },
+        totalCate:(state,action)=>{
+            state.meta=action.payload
+        },
+    },
+    
+    extraReducers:(builder)=>{
+        builder
+            .addCase(getCategories.fulfilled,(state,action)=>{
+                state.list.push(action.payload.items);
```

Comment:
em sửa lại cách viết như này, tương tự file taskSlice em cũng sửa lại nhé
```suggestion
                state.list = action.payload.items;
```